File: MatematicaApp/src/main/java/com/sergiocruz/Matematica/activity/AboutActivity.java
package com.sergiocruz.Matematica.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;
import android.widget.TextView;

import com.sergiocruz.Matematica.R;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        ((TextView) findViewById(R.id.aboutTitle))
                .setText(getString(R.string.app_long_description)+getString(R.string.app_version_name));

        ((TextView) findViewById(R.id.version))
                .setText(getString(R.string.app_version_description) + " " + getString(R.string.app_version_name) + "\n"
                + getString(R.string.app_version_date));
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();

        //noinspection SimplifiableIfStatement
        if (id == android.R.id.home) {
            // finish the activity
            onBackPressed();
            return true;
        }

        return super.onOptionsItemSelected(item);
    }
}

File: MatematicaApp/src/main/java/com/sergiocruz/Matematica/activity/AboutActivity.kt
package com.sergiocruz.Matematica.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem

import com.sergiocruz.Matematica.R

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        val id = item.itemId

        if (id == android.R.id.home) {
            // finish the activity
            onBackPressed()
            return true
        }

        return super.onOptionsItemSelected(item)
    }
}
